### ISSUE TITLE
Redactor - insert image doesn't work

### ISSUE THREAD

**Issue by dannyuk1982** [statamic/v2-hub#1154]
Fieldset yaml:

```
          paragraphs:
            type: redactor
            display: ' '
            settings: Standard
            container: attachments
            folder: pages
```

I can insert images via the asset button, they appear like this:

<img width="255" alt="screen shot 2016-12-09 at 14 45 49" src="https://cloud.githubusercontent.com/assets/7669885/21053072/b1377e46-be1f-11e6-92b5-2c3a0834eb1b.png">  

&nbsp;

The generated HTML is:

```
      <p><img src="assets/attachments/pages/AdobeStock_106751477_WM.jpeg" alt="">
```

looks like it needs a `/` (or correct non-relative route) before the `assets` and from the CP, this obviously doesn't link to anything - unless I am missing something

**Comment by dannyuk1982** [comment]
I don't have the src files, but in the compacted version, line 5 there is a bit:

`redactor("insert.html",'<img src="'+e+'" alt="'+t+'" />')},appendImage:function(e,t){var n=$(this.$els.redactor)`

change this to:
`redactor("insert.html",'<img src="/'+e+'" alt="'+t+'" />')},appendImage:function(e,t){var n=$(this.$els.redactor)`

i.e. add in a `/` after `<img src="` and it fixes it. There must be a more elegant way to do it using site root

**Comment by jackmcdade** [comment]
This will be fixed in 2.2!
